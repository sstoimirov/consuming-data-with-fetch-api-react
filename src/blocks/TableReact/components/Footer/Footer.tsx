import * as React from "react";

export class Footer extends React.PureComponent<{}, {}>{
    render() {
        return (
            <footer className="footer-container">
                <div className="footer-copyrights">2003 - 2011 Copyright 10bet.com Ltd. All rights reserved.</div>
                <div className="footer-links-elements">
                    <div>Home</div>
                    <div>Terms of use</div>
                    <div>Responsible gaming</div>
                    <div>Contact us</div>
                </div>
            </footer>
        )
    }
}