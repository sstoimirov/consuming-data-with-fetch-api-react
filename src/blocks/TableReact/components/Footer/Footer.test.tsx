import * as React from "react";
import { shallow,configure } from "enzyme";
import { Footer } from "./Footer";
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })

describe("Footer",()=>{
    it("Renders correctly", ()=>{
        const comp = shallow(<Footer/>);
        expect(comp.debug()).toMatchSnapshot();
    })
})