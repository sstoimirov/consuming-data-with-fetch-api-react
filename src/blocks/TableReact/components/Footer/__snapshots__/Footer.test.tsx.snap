// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Renders correctly 1`] = `
"<footer className=\\"footer-container\\">
  <div className=\\"footer-copyrights\\">
    2003 - 2011 Copyright 10bet.com Ltd. All rights reserved.
  </div>
  <div className=\\"footer-links-elements\\">
    <div>
      Home
    </div>
    <div>
      Terms of use
    </div>
    <div>
      Responsible gaming
    </div>
    <div>
      Contact us
    </div>
  </div>
</footer>"
`;
